src/it/junit/filter-tag-expressions-scenario/src/test/resources/features/feature1.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Feature1
+@test_for_england
+
+  Scenario: Generate Junit Runner for each feature file
```

Comment:
The tag should be right above the scenario line.

Reply:
Same for the other integration test

Reply:
true... i will fix it